airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -230,7 +230,7 @@ def __init__(
         self.schema = schema
         self.authenticator = authenticator
         self.session_parameters = session_parameters
-        self.query_ids = []
+        self.query_ids: List[str] = []
```

Comment:
Strange that it's not inferred automatically here. But it does not hurt.

Reply:
But maybe you can remove it since the docstring needs to be fixed.

Reply:
Yeah, unfortunately, it doesn't infer automatically. 
I tried without this hint but mypy doesn't like it.